simpleclient_hotspot/src/test/java/io/prometheus/client/hotspot/ThreadExportsTest.java

```diff
@@ -0,0 +1,66 @@
+package io.prometheus.client.hotspot;
+
+import io.prometheus.client.Collector;
+import io.prometheus.client.CollectorRegistry;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.lang.management.MemoryMXBean;
+import java.lang.management.MemoryPoolMXBean;
+import java.lang.management.MemoryUsage;
+import java.lang.management.ThreadMXBean;
+import java.util.Arrays;
+import java.util.Enumeration;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+
+public class ThreadExportsTest {
+
+  private ThreadMXBean mockThreadsBean = Mockito.mock(ThreadMXBean.class);
+  private CollectorRegistry registry = new CollectorRegistry();
+  private ThreadExports collectorUnderTest;
+
+  @Before
+  public void setUp() {
+    when(mockThreadsBean.getThreadCount()).thenReturn(300);
+    when(mockThreadsBean.getDaemonThreadCount()).thenReturn(200);
+    when(mockThreadsBean.getPeakThreadCount()).thenReturn(301);
+    when(mockThreadsBean.getTotalStartedThreadCount()).thenReturn(503L);
+    collectorUnderTest = new ThreadExports(mockThreadsBean).register(registry);
+  }
+
+  @Test
+  public void testThreadPools() {
+    assertEquals(
+            200L,
+            registry.getSampleValue(
+                    "jvm_threads",
+                    new String[]{"type"},
```

Comment:
This is used a few times, could benefit from being a const or defined in the set up. Depends if its necessary